modules/rostests/apitests/user32_dynamic/load.c

```diff
@@ -18,104 +19,129 @@ START_TEST(load)
     PPEB Peb = NtCurrentPeb();
 
     /* Before init */
+    SetLastError(0xdeadbeef);
     hUser32 = GetModuleHandleW(L"user32");
-    ok(hUser32 == NULL, "hUser32 = %p\n", hUser32);
+    Error = GetLastError();
+
+    if (hUser32 != NULL)
+    {
+        win_skip("user32.dll is already loaded since Vista\n");
```

Comment:
Any clue why? Does `msvcrt` import it maybe? If we know how it gets pulled in, we might be able to fix that.